app/resources/v1/announcement_resource.rb

```diff
@@ -1,10 +1,10 @@
-class Basechurch::V1::AnnouncementResource < JSONAPI::Resource
+class V1::AnnouncementResource < JSONAPI::Resource
   attributes :description, :position, :url
 
   has_one :bulletin
   has_one :post
 
-  model_name 'Basechurch::Announcement'
+  model_name 'Announcement'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.